react/store/createReturnRequest/components/loaders/ReturnDetailsLoader.tsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react'
+import type { FunctionComponent } from 'react'
+import { SkeletonPiece, BaseLoading } from 'vtex.my-account-commons'
+import type { OrderToReturnSummary } from 'vtex.return-app'
+import { PageBlock } from 'vtex.styleguide'
+import { FormattedMessage } from 'react-intl'
+
+interface Data {
+  loading: boolean
+  data: OrderToReturnSummary | undefined
+}
+
+interface Props {
+  data: Data
+}
+
+export const ReturnDetailsLoader: FunctionComponent<Props> = ({ data }) => {
+  const headerConfig = {
+    namespace: 'vtex-account__return-details',
+    title: (
+      <FormattedMessage id="store/return-app.return-order-details.page-header.title" />
+    ),
+    titleId: 'store/return-app.request-return.page.header"',
+    backButton: {
+      titleId: 'store/return-app.return-order-details.page-header.link',
+      path: '#/my-returns/add',
+    },
+  }
```

Comment:
Is there a way to apply the same headerConfig to the ReturnDetails component? I feel they are doing the same thing, but there we are using a PageHeader component. Maybe we can define a object with the `title` component, `linkLabel` and `to` address and use from it. What do you think?

Reply:
I like the idea of the object!